test/integration/test_create_dss_docker_image.py

```diff
@@ -1,33 +1,42 @@
 import docker
-import logging
 import pytest
 import requests
 import time
 
+from datetime import datetime, timedelta
 from exasol.ds.sandbox.lib.dss_docker import DssDockerImage
-from datetime import datetime
+from exasol.ds.sandbox.lib.logging import set_log_level
+from exasol.ds.sandbox.lib import pretty_print
 
 
 @pytest.fixture(scope="session")
-def dss_docker_container():
-    timestamp = f'{datetime.now().timestamp():.0f}'
+def dss_docker_image():
+    set_log_level("info")
     testee = DssDockerImage(
         "my-repo/dss-test-image",
-        version=f"{timestamp}",
+        version=f"{DssDockerImage.timestamp()}",
         publish=False,
-        log_level=logging.INFO,
-    )
-    print(
-        "\n- Using"
-        f' Docker container {testee.container_name}'
-        f' with image {testee.image_name}'
+        keep_container=False,
     )
+    # print(
+    #     "\n- Using"
+    #     f' Docker container {testee.container_name}'
+    #     f' with image {testee.image_name}'
+    # )
     testee.create()
+    try:
+        yield testee
+    finally:
+        docker.from_env().images.remove(testee.image_name)
+
+
+@pytest.fixture(scope="session")
```

Comment:
To make this a session fixture needs to be discussed, because you might actually want a new container,  in case your test messes up the container. Starting a new container also should be a big time investment.